content/maintainers-guide/how-to-become-a-committer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How to become a committer
+weight: 12
+---
+
+# How to become a committer
+
+In the PrestaShop project, committer are contributors with partial merge rights. 
```

Comment:
Oh actually one comment on this. Should the commiters have merge rights as well? Or only approval rights?

We could keep merge rights for maintainers only as a safeguard. Since anyway at least one approval from a maintainer is required, it makes sense to leave the merge rights to maintainers only? What do you think?

Reply:
Interesting. In theory anybody could merge a PR as long as it matches our criteria: 2 approvals (at least 1 maintainer), CI passes, verified by QA, and any item like wording or UX validated if eligible so maybe we could also open merge rights.

But I think GitHub will not allow us to do this. GitHub requests a group to be "code owner" and I think this is the people able to push and merge on the branch.